pkg/controller/node.go

```diff
@@ -80,7 +80,7 @@ func (c *Controller) moveMasterPodsOffNode(node *v1.Node) {
 	opts := metav1.ListOptions{
 		LabelSelector: labels.Set(c.opConfig.ClusterLabels).String(),
 	}
-	podList, err := c.KubeClient.Pods(c.opConfig.Namespace).List(opts)
+	podList, err := c.KubeClient.Pods("").List(opts)
```

Comment:
Please comment on the effect of using the empty string aka `""` as a namespace name. Does it mean "listen to all namespaces" ? I traced it down to the [`Namespace` method](https://github.com/kubernetes/client-go/blob/4b76cf9824ec474ca9b122449fee23807a51e786/rest/request.go#L243) of the k8s go-client, and still wasn't quite sure.